### ISSUE TITLE
Geographic heatmap feature

### ISSUE THREAD

**Issue by brylie** [Gapminder/vizabi#490]
# User Story
```
As a researcher
I would like to view geographic clustering of one or more variables
so that I can visualize geographic distribution of data over time
```

# Feature request
Add a geographic display of data with base map. Overlay the base map with a data clustering layer, such as a [geographical heatmap](https://en.wikipedia.org/wiki/Heat_map).

![Example geographical heatmap](https://en.wikipedia.org/wiki/Heat_map#/media/File:WOA09_sea-surf_SAL_AYool.png)

# Related tools/libraries
* [OpenLayers 3](http://openlayers.org has robust [heatmap](http://openlayers.org/en/master/apidoc/ol.layer.Heatmap.html) support

**Comment by angieskazka** [comment]
Yes! We welcome volunteers who would like to develop this app for http://vizabi.org/
